File: app/src/main/java/com/tungnui/abccomputer/model/ProductReview.java
package com.tungnui.abccomputer.model;

/**
 * Created by Nasir on 3/29/17.
 */


public class ProductReview {
    public int id;
    public String createdDateTime;
    public float rating;
    public String review;
    public String name;
    public String email;

    public ProductReview(int id, String createdDateTime, float rating, String review, String name, String email) {
        this.id = id;
        this.createdDateTime = createdDateTime;
        this.rating = rating;
        this.review = review;
        this.name = name;
        this.email = email;
    }
}

File: app/src/main/java/com/tungnui/dalatlaptop/models/ProductReview.kt
package com.tungnui.dalatlaptop.models

/**
 * Created by thanh on 07/11/2017.
 */
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ProductReview (
    @SerializedName("id")
    @Expose
    var id: Int? = null,
    @SerializedName("date_created")
    @Expose
    var dateCreated: String? = null,
    @SerializedName("date_created_gmt")
    @Expose
    var dateCreatedGmt: String? = null,
    @SerializedName("review")
    @Expose
    var review: String? = null,
    @SerializedName("rating")
    @Expose
    var rating: Int? = null,
    @SerializedName("name")
    @Expose
    var name: String? = null,
    @SerializedName("email")
    @Expose
    var email: String? = null,
    @SerializedName("verified")
    @Expose
    var verified: Boolean? = null
)